internal/parser/parser.go

```diff
@@ -1054,6 +1052,68 @@ func (p *parser) parseParenIdentExpression(lparen token.Pos, key *ast.Identifier
 	}
 }
 
+func (p *parser) parseObjectBody() *ast.ObjectExpression {
+	switch _, tok, _ := p.peek(); tok {
+	case token.IDENT:
+		ident := p.parseIdentifier()
+		return p.parseObjectBodySuffix(ident)
+	case token.STRING:
+		str := p.parseStringLiteral()
+		properties := p.parsePropertyListSuffix(str)
+		return &ast.ObjectExpression{
+			Properties: properties,
+		}
+	default:
+		return &ast.ObjectExpression{
+			Properties: p.parsePropertyList(),
+		}
+	}
+
+}
+func (p *parser) parseObjectBodySuffix(ident *ast.Identifier) *ast.ObjectExpression {
+	switch _, tok, lit := p.peek(); tok {
+	case token.IDENT:
+		if lit != "with" {
```

Comment:
Should we be adding `with` as a keyword?

If we don't add `with` as a keyboard, can you somehow fall through to the default so it produces an invalid AST? I think if we see `{a b}` we can assume that this was a mistake for `{a: b}` which the default case handles.

Reply:
added as a keyword.